app/Http/Controllers/UserController.php

```diff
@@ -14,6 +14,8 @@ class UserController extends Controller
 {
     use PasswordValidationRules;
 
+    private string $userMaxColumnSize = 'max:255';
```

Comment:
same comment as with the redirectTo @alexstojda

Reply:
@alecadub You can mark it as a false positive in Sonarcloud and undo the change